druid-shell/src/window.rs

```diff
@@ -210,6 +210,18 @@ pub enum MouseButton {
     X2,
 }
 
+impl MouseButton {
+    /// Returns `true` if this is the left mouse button.
+    pub fn is_left(self) -> bool {
```

Comment:
Maybe `#[inline]` because this the comparison is cheaper than the function call?